maskrcnn_benchmark/utils/checkpoint.py

```diff
@@ -79,6 +79,7 @@ def get_checkpoint_file(self):
         try:
             with open(save_file, "r") as f:
                 last_saved = f.read()
+                last_saved = last_saved.replace("\n", "")
```

Comment:
what about using `.strip()` instead, would that be more robust (for example, on Windows)?